sa-iso-15919-itrans.mim

```diff
@@ -105,8 +105,6 @@ Also, see: http://en.wikipedia.org/wiki/Devanagari_transliteration.")
 ("a''" "á")
 ("i''" "í")
 ("u''" "ú")
-("uu''" "ṹ")
```

Comment:
इदम् इह सम्यग् एव +अस्तीति मन्ये - U+1E79 : LATIN SMALL LETTER U WITH TILDE AND ACUTE । unicode इति विशिष्टां सङ्केतग्रामबोधिकां सङ्ख्यां सूचयति चेत्, तद्-उपयोगो वरः - यतः, तत्प्रदर्शने ऽक्षरमुख(font)-निर्मातारो विशिष्य श्राम्येयुः।

Reply:
ástu aivám

Reply:
Astu - tarhi yathApUrvam iha kRtvA nudatu.

Reply:
ऒह्! अत्र ~ अस्ति!! न मया पूर्वं लक्षितम्। भवता साधु निष्कासितम्।

Reply:
nudā́mi